src/utils/activeModal.js

```diff
@@ -0,0 +1,9 @@
+import React, { useContext } from "react";
+
+const activeModalContext = React.createContext({
```

Comment:
`librarySelectionContext` and `activeModalContext` should be in **UpperCase** because context variables act like a `component` in the markup